config/patches/ohai/fix_default_route.patch

```diff
@@ -0,0 +1,62 @@
+From 2d43bf4238d60bb3e000c01eb9bbe4911cbc14ac Mon Sep 17 00:00:00 2001
+From: Bao Nguyen <bao@fastly.com>
+Date: Thu, 30 Apr 2015 16:53:16 -0700
+Subject: [PATCH 1/2] at fastly we want default route only
+
+---
+ lib/ohai/plugins/linux/network.rb | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/lib/ohai/plugins/linux/network.rb b/lib/ohai/plugins/linux/network.rb
+index 3d80a05..bcb3d3b 100644
+--- a/lib/ohai/plugins/linux/network.rb
++++ b/lib/ohai/plugins/linux/network.rb
+@@ -200,7 +200,10 @@ Ohai.plugin(:Network) do
+         #    the routing table source field.
+         # 3) and since we're at it, let's populate some :routes attributes
+         # (going to do that for both inet and inet6 addresses)
+-        so = shell_out("ip -o -f #{family[:name]} route show")
++        #
++        # 4) At Fastly we only want the default route because the main route
++        # table has over 2millions+ routes.
++        so = shell_out("ip -o -f #{family[:name]} route show 0/0")
+         so.stdout.lines do |line|
+           line.strip!
+           Ohai::Log.debug("Parsing #{line}")
+--
+2.3.2
+
+
+From a0029a98139dba1b779d08c20de7148fc114f4c3 Mon Sep 17 00:00:00 2001
+From: Bao Nguyen <bao@fastly.com>
+Date: Fri, 1 May 2015 11:17:24 -0700
+Subject: [PATCH 2/2] we'll want to collect v6 address too
+
+---
+ lib/ohai/plugins/linux/network.rb | 10 +++++++++-
+ 1 file changed, 9 insertions(+), 1 deletion(-)
+
+diff --git a/lib/ohai/plugins/linux/network.rb b/lib/ohai/plugins/linux/network.rb
+index bcb3d3b..3509232 100644
+--- a/lib/ohai/plugins/linux/network.rb
++++ b/lib/ohai/plugins/linux/network.rb
+@@ -203,7 +203,15 @@ Ohai.plugin(:Network) do
+         #
+         # 4) At Fastly we only want the default route because the main route
+         # table has over 2millions+ routes.
+-        so = shell_out("ip -o -f #{family[:name]} route show 0/0")
++
++        # making sure we both the v4 and v6 default address
++        so = ""
```

Comment:
This should be simplified so that only the argument is conditional:

```ruby
route = family[:name] == 'inet6' ? '::0/0' : '0/0'
so = shell_out("ip -o -f #{family[:name]} route show #{route}")
```